airb-and-b/src/App.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
```

Comment:
Since you have used react version 18, you don't need to import the react, react handle it without importing it, you should remove it.